src/sbt-test/skip-windows/caching/build.sbt

```diff
@@ -7,7 +7,8 @@ inThisBuild(
     scalacOptions += "-Yrangepos",
     scalacOptions += "-Ywarn-unused", // for RemoveUnused
     scalafixCaching := true,
-    scalafixDependencies += "com.geirsson" %% "example-scalafix-rule" % "1.2.0"
+    scalafixDependencies += "com.nequissimus" %% "sort-imports" % "0.5.3",
```

Comment:
Is there a reason you can use the example scala rule? Typically you want to avoid depending on 3rdparty dependencies like this in your tests so that you have more control over everything.

Reply:
I mentioned the reason in [the commit message](https://github.com/scalacenter/sbt-scalafix/pull/121/commits/d6124f0445339d8ca12c968263e968b15d204e68). I wanted to move from `com.geirsson` to `ch.epfl.scala` everywhere, but couldn't do it here as I needed 2 revisions (see usage of `set scalafixDependencies` in the corresponding `test`). But I agree it does mot make sence to move from one third party (even if it's yours) to another. I'll revert changes here in that specifc reference and in `test`.